start.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-npm install;
-npm install -g bower
+sudo npm install
```

Comment:
I'm not sure we should put sudo in here. If we need to run the command as root can't we just run sudo ./start.sh